src/index.js

```diff
@@ -0,0 +1,178 @@
+let temperatureLabel = null;
+let temperatureIncreaseButton = null;
+let temperatureDecreaseButton = null;
+let getRealtimeTemperatureButton = null;
+let skyImage = null;
+let landscapeImage = null;
+let cityNameInput = null;
+let cityNameResetButton = null;
+let skySelectValue = null;
+let gardenBackground = null;
+let temperatureValue = null;
+let defaultCityName = null;
+
+const showTemp = () => {
+    temperatureLabel.textContent = temperatureValue;
+}; 
+
+const showCity = () => {
+    cityNameLabel.textContent = defaultCityName;
+    cityNameInput.value = defaultCityName;
+}; 
+
+const setTempLabelColor = (temperature) => {
+    if (temperature >= 80) {
+        return temperatureLabel.setAttribute("class", "red");
+    } else if (temperature >= 70) {
+        return temperatureLabel.setAttribute("class", "orange");
+    } else if (temperature >= 60) {
+        return temperatureLabel.setAttribute("class", "yellow");
+    } else if (temperature >= 50) {
+        return temperatureLabel.setAttribute("class", "green");
+    } else {
+        return temperatureLabel.setAttribute("class", "teal");
+    }
+};
+
+const landscapeImages = {
+    80: "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+    70: "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+    60: "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+    50: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲",
+};
+
+const skyImages = {
+    Sunny: "☁️ ☁️ ☁️ ☀️ ☁️ ☁️",
+    Cloudy: "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+    Rainy: "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+    Snowy: "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨",
+};
```

Comment:
For the sky values, a dictionary works well since we can look up the options directly. However, we could iterate over the dictionary keys (or use a tuple list similar to that described for the landscape) in order to generate the lookup dictionary, and populate the options in the select. This approach can be useful if the options are to be loaded from a backend database!